### ISSUE TITLE
Shell and Shell Panel - Provide a valid label for form fields - (2036570582)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5600]
### Module: 
 29 Shell and Shell Panel 

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570582 

### Severity: 
 10 

### Description: 
 [Pattern: Button- Disabled non meaningful button label.]

[Issue]
The save action button, this includes the "Disabled" word in the label. It is rendered as "Disabled save action, for display purposes only". This is not meaningful.

[Occurrences]
29 Shell and Shell Panel
30 Shell Center Row

[User Impact]
Screen reader users may not fully understand form controls that do not provide meaningful labels.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="true" aria-label="Disabled save action, for display purposes only" class="button button--text-visible" disabled=""><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="save" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container text-container--visible">Save action, disabled</div></button>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with internal text, an aria-label attribute, or an aria-labelledby attribute. Good accessible names are both concise and descriptive.
- Remove the text "Disabled" from the accessible name. 

[Compliant Code Example] 
 ```
<button aria-busy="false" aria-disabled="true" aria-label="save action, for display purposes only" class="button button--text-visible" disabled=""><div aria-hidden="true" class="icon-container"><calcite-icon aria-hidden="true" icon="save" scale="s" calcite-hydrated=""></calcite-icon><div class="slot-container"><slot></slot></div></div><div class="text-container text-container--visible">Save action, disabled</div></button>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.
